Update troubleshoot-visualizer.adoc (#78)
mulesoft/docs-visualizer: modules/ROOT/pages/troubleshoot-visualizer.adoc

=== Why do some Mule apps and APIs deployed on premises show incorrect or incomplete metrics and connection information?

include::partial$include-mule-domain-support.adoc[]

> === Why can't I see metadata from an HTTP requester?

Author: ```suggestion
=== Why don't I see metadata from an HTTP requester?
```

> To avoid high cardinality, Anypoint Visualizer does not support outbound traffic that uses IP addresses or localhost for validation.

Author: ```suggestion
To avoid high graph cardinality, Anypoint Visualizer does not show outbound traffic to IP addresses or localhost.
```
